cmd/kar-controllers/app/server.go

```diff
@@ -41,8 +41,11 @@ func Run(opt *options.ServerOption) error {
 
 	neverStop := make(chan struct{})
 
-	queuejobctrl := queuejob.NewQueueJobController(config)
-	queuejobctrl.Run(neverStop)
+	//queuejobctrl := queuejob.NewQueueJobController(config)
+	//queuejobctrl.Run(neverStop)
```

Comment:
We need to keep QueueJob controller here.

Reply:
@scostache Could you comment on the impl. here? ^^^